Simplenote/NSWindowFrameAutosaveName+Simplenote.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+
+// MARK: - NSWindow.FrameAutosaveName
+//
+extension NSWindow.FrameAutosaveName {
+
+    /// Main Window Autosave Name
+    ///
+    static var mainWindow: NSWindow.FrameAutosaveName {
+        "SimplenoteMainWindow"
+    }
```

Comment:
What about simplifying this down to
```suggestion
    static var mainWindow: NSWindow.FrameAutosaveName = "SimplenoteMainWindow"
```

Reply:
Thank you!!